cmd/kwil-cli/cmds/common/authinfo.go

```diff
@@ -84,6 +85,10 @@ func getDomain(target string) (string, error) {
 		return "", fmt.Errorf("target is empty")
 	}
 
+	if !(strings.HasPrefix(target, "http://") || strings.HasPrefix(target, "https://")) {
```

Comment:
For 0.7 or v0.6, this should work, since we require scheme https://docs.kwil.com/docs/kwil-cli/configuration#available-configs. 

For v0.8, seems this could break?

Reply:
you got a pretty handy util function here https://github.com/kwilteam/kwil-db/pull/703/files#diff-7a8b6c70656b28a8bba3d014ad27d1a48efe27f72f184939a4b9d7b5d2313a84R72, I think it will help.

Reply:
With latest update to that PR, `cfg.Provider` must be a proper URL now.  Another reason to keep that convention I figure.

Reply:
The persistened `cfg.Provider` could still be without scheme, am I right?

Reply:
Not any more.  It's an error:

```
$  ./kwil-cli database list
Error: parse url: parse "127.0.0.1:8484": first path segment in URL cannot contain colon
parse url: parse "127.0.0.1:8484": first path segment in URL cannot contain colon
$  cat ~/.kwil-cli/config.json 
{"provider":"127.0.0.1:8484"}
```

I think we had that same constraint in v0.7?

Reply:
We have some unpleasant constraint, like this :
```
.build/kwil-cli database list --kwil-provider localhost:8080                                                   
Error: wrap client: chain_info: Get "localhost:8080/api/v1/chain_info": unsupported protocol scheme "localhost"
wrap client: chain_info: Get "localhost:8080/api/v1/chain_info": unsupported protocol scheme "localhost"
```

It'll get error when create client, that's before `getDomain`, so I guess we're fine